### PR TITLE
Added menu

### PR DESCRIPTION
**Pull request by alansouzati**

### CODE REVIEW

File: src/js/components/Menu/doc.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { schema, PropTypes } from 'react-desc';
+
+const VERTICAL_ALIGN_OPTIONS = ['top', 'bottom'];
+const HORIZONTAL_ALIGN_OPTIONS = ['right', 'left'];
+
+export default Menu => schema(Menu, {
+  description: `Presents a list of choices within a drop down behind a control that
```
Comment (ericsoderberghp) [inline src/js/components/Menu/doc.js]: 'behind' -> 'via'?

File: src/js/components/Menu/doc.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  opens it.`,
+  usage: `import { Menu } from 'grommet';
+  <Menu/>`,
+  props: {
+    color: [
```
Comment (ericsoderberghp) [inline src/js/components/Menu/doc.js]: Should this be `background`, to align more with `Box`?

File: src/js/components/Menu/doc.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      `Where to place the drop down. The keys correspond to a side of the drop down.
+      The values correspond to a side of the control. For instance,
+      {left: 'left', top: 'bottom'} would align the left edges and the top of 
+      the drop down to the bottom of the control. At most one of left or right and
+      one of top or bottom should be specified.`,
+    ],
+    icon: [
+      PropTypes.node,
+      `Indicates that the menu should be collapsed and the icon shown as a 
+      control top open it.`,
```
Comment (ericsoderberghp) [inline src/js/components/Menu/doc.js]: 'top' -> 'to'

File: src/js/components/Menu/doc.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      }),
+      `Where to place the drop down. The keys correspond to a side of the drop down.
+      The values correspond to a side of the control. For instance,
+      {left: 'left', top: 'bottom'} would align the left edges and the top of 
+      the drop down to the bottom of the control. At most one of left or right and
+      one of top or bottom should be specified.`,
+    ],
+    icon: [
+      PropTypes.node,
+      `Indicates that the menu should be collapsed and the icon shown as a
```
Comment (ericsoderberghp) [inline src/js/components/Menu/doc.js]: Doesn't the menu always start collapsed?

File: src/js/components/Menu/doc.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+      `Indicates that the menu should be collapsed and the icon shown as a 
+      control top open it.`,
+    ],
+    items: [
+      PropTypes.arrayOf(PropTypes.shape({
+        label: PropTypes.string,
+        onClick: PropTypes.func,
+        icon: PropTypes.node,
+      })),
+      'Menu items to be placed inside the drop down. All button properties should be supported.', {
```
Comment (ericsoderberghp) [inline src/js/components/Menu/doc.js]: "should be" -> "are also"?

Comment (ericsoderberghp) [reply src/js/components/Menu/doc.js]: Also, I assume one of `label` or `icon` must be provided.

File: src/js/components/Menu/doc.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    ],
+    icon: [
+      PropTypes.node,
+      `Indicates that the menu should be collapsed and the icon shown as a 
+      control top open it.`,
+    ],
+    items: [
+      PropTypes.arrayOf(PropTypes.shape({
+        label: PropTypes.string,
+        onClick: PropTypes.func,
```
Comment (ericsoderberghp) [inline src/js/components/Menu/doc.js]: Why is `onClick` called out here, if all `Button` properties are passed through. We don't list other `Button` properties.

File: src/js/components/Box/Box.js [hunk 7]
```diff
@@ -27,7 +27,7 @@ class Box extends Component {
   };
 
   getChildContext() {
-    const { grommet } = this.context;
+    const { grommet = {} } = this.context;
```
Comment (ericsoderberghp) [inline src/js/components/Box/Box.js]: Given `grommet` is required, do we need this initializer?

Comment (alansouzati) [reply src/js/components/Box/Box.js]: nope :)

**Pull request merged**